stellar-auth/pkg/utils/password_validation.go

```diff
@@ -34,8 +46,43 @@ func (e *ValidatePasswordError) FailedValidations() map[string]string {
 	return e.FailedValidationsMap
 }
 
+type PasswordValidator struct {
+	commonPasswordsList map[string]bool
+}
+
+func NewPasswordValidator() (*PasswordValidator, error) {
```

Comment:
nit:  Entirely up to you, but I would change this method name to `GetPasswordValidatorInstance()` or better name. 

`NewPasswordValidator()` is misleading because we're not creating a new instance every time we're calling this method.